docker-compose_development.yml

```diff
@@ -323,6 +323,37 @@ services:
         target: /home/mig/state/mig_system_run
     command: /app/docker-entry.sh
 
+  migrid-lustre-quota:
```

Comment:
Hmmm, I'm not convinced we want Lustre in this 'toy' setup at all.
https://docker-migrid.readthedocs.io/en/latest/sections/getting-started/introduction.html#example-deployments
Please also see comment about production_gdp below.